packages/main/src/ListItem.ts

```diff
@@ -50,6 +50,16 @@ type PressEventDetail = {
 	key: string,
 }
 
+type AccInfo = {
+	role: string;
+	ariaExpanded: boolean | undefined;
+	ariaLevel: number | undefined;
+	ariaLabel: string;
+	ariaLabelRadioButton: string;
+	ariaSelectedText: string | undefined;
+	ariaHaspopup: HasPopup | undefined;
```

Comment:
If you mark these properties as optional with ? second undefined type is redudant